### ISSUE TITLE
NiceHashMiner2 Don´t start with cpu only machine.

### ISSUE THREAD

**Issue by meepominer** [nicehash/NiceHashMiner2#67]
NiceHashMiner2 doesn´t start on a pc for cpu only mining, there is no amd or nvidia card installed on it.

**Comment by blockbasti** [comment]
What CPU do you have? OS? Did older versions work. Maybe try the old NHM.

**Comment by meepominer** [comment]
Cpu: 3rd Gen I5
OS: Windows 10
The older version of NihceHash do work for cpu only mining.

The problems seems to be that NHM2 always opens excavator.exe and maybe when there is no gpu detected it hangs up and don´t let NHM2 start.

**Comment by S74nk0** [comment]
@meepominer Is this issue is with v2.0.0.6?

**Comment by meepominer** [comment]
Yes it is an issue with v2.0.0.6, apparently excavator.exe doesn´t load so NHMiner2 doesn´t load also.
